docs/reference/sdk/interfaces/core.ExecutionContext.md

```diff
@@ -67,7 +67,7 @@ ___
 
 ### sync
 
-• `Optional` `Readonly` **sync**: [`Sync`](core.Sync.md)
+• `Optional` `Readonly` **sync**: [`Sync`](core.Sync.md)<`unknown`, `unknown`\>
```

Comment:
My only concern is whether impacts like this on the generated documentation is too distracting. If @ekoleda-codaio signs off, I'm happy.

Reply:
Not too distracting, and I don't think many developers consult the reference docs. LGTM.